MockKit/MockCGMManager.swift

```diff
@@ -299,7 +299,7 @@ public final class MockCGMManager: TestingCGMManager {
         return MockCGMManager.managerIdentifier
     }
     
```

Comment:
Most of our other services follow the convention <Name CGM>. Applying that pattern here to help with our sorted list of CGMs.

Reply:
I think in this case simulator is the noun and CGM is the adjective, whereas in other cases CGM is the noun, so I think it should stay in this order.